File: Programlar/Task 1/1/1.py
n = int(input())
eks = 0
while n:
    eks = eks*10 + n%10
    n//=10
else:
    if n == eks: print("Yes")
    else: print("No")

# Verilmiş ədədin polindrom olub-olmadığını yoxlayan program

File: Programlar/Task 1/1/1.cpp
#include<iostream>
using namespace std;
int main(){
	int n, temp, eks=0;
	cin>>n;
	temp = n;
	while(temp){
		eks = eks*10 + temp%10;
		temp/=10;
	}
	if(n==eks) cout<<"Yes";
	else cout<<"No";
	return 0;
}

//Verilmis ededin polindrom olub olmadigini yoxlayan program
